plugins/inputs/linux_sysctl_fs/linux_sysctl_fs.go

```diff
@@ -17,7 +17,7 @@ import (
 //go:embed sample.conf
 var sampleConfig string
 
-// https://www.kernel.org/doc/Documentation/sysctl/fs.txt
+// SysctlFS - see https://www.kernel.org/doc/Documentation/sysctl/fs.txt
```

Comment:
Can we please remove this comment as it's not documenting the plugin struct IMO...